Cargo.toml

```diff
@@ -28,7 +28,8 @@ state = { path="./state", default-features = false }
 members = [
     "evm",
     "ewasm",
-    "state"
+    "state",
+    "example"
```

Comment:
`example` should not be in with members.

Reply:
It seems must be contained in members, Reason as following:
```
wpf@WPF:~/working/github/ouwenkg/cita-vm/example$ cargo run
error: current package believes it's in a workspace when it's not:
current:   /home/wpf/working/github/ouwenkg/cita-vm/example/Cargo.toml
workspace: /home/wpf/working/github/ouwenkg/cita-vm/Cargo.toml

this may be fixable by adding `example` to the `workspace.members` array of the manifest located at: /home/wpf/working/github/ouwenkg/cita-vm/Cargo.toml
```

Reply:
Ok